checker/src/main/java/org/checkerframework/checker/mustcall/MustCallVisitor.java

```diff
@@ -73,6 +75,35 @@ public Void visitReturn(ReturnTree tree, Void p) {
     return super.visitReturn(tree, p);
   }
 
+  @Override
+  public Void visitAssignment(AssignmentTree tree, Void p) {
+    // The following code implements the following rule:
+    //  * It is always safe to assign a MustCallAlias parameter of a constructor
+    //    to an owning field of the containing class.
+    // It is necessary to special case this because MustCallAlias is translated
+    // into @PolyMustCall.
```

Comment:
I don't fully understand this comment, though perhaps it's because I'm unfamiliar with how `@PolyMustCall` is implemented